onnx/defs/tensor/defs.cc

```diff
@@ -381,7 +381,7 @@ Example 2:
             // handle some simple cases.
             if (ctx.getInputType(0)->tensor_type().shape().dim((int)i).has_dim_value() &&
                 starts[j] >= 0 && ends[j] >= 0) {
-              auto newval = std::min(ctx.getInputType(0)->tensor_type().shape().dim((int)i).dim_value(), ends[j]) - starts[j];
+              auto newval = std::min((int64_t)ctx.getInputType(0)->tensor_type().shape().dim((int)i).dim_value(), ends[j]) - starts[j];
```

Comment:
Why not add a template argument to std::min? It would be safer is someone changed the type of "ends" in the latter.